### PR TITLE
Video stream

### PR DESCRIPTION
**Pull request by avalmas-programize**
Related Issue: https://github.com/aqualinkorg/aqualink-app/issues/571

If stream is present it shows up in the reef info cards and it forces them to follow the 16:9 ratio
Stream auto-plays on load of the page (only for **desktop** devices)

Designs:
| desktop | iPad | iPhone 6/7/8 |
|---|---|---|
| desktop | ipad | iphone |

### CODE REVIEW

**Comment by ericboucher** [comment]
Nice work. Let's remove the grey side round up the video instead.

Screen Shot 2021-06-17 at 2 14 04 PM

File: packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx [hunk 1]
```diff
@@ -34,13 +33,13 @@ const FeaturedMedia = ({
...
       <Card className={classes.card}>
         <CardContent className={classes.content}>
-          <ReactPlayer
-            height="100%"
-            width="100%"
-            playing
-            muted
-            light
-            url={`${url}`}
+          <CardMedia
```
Comment (ericboucher) [inline packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx:36]: what's the rationale for this change? (just curious)

File: packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx [hunk 2]
```diff
@@ -12,7 +12,6 @@ import {
   Theme,
 } from "@material-ui/core";
 import { Link } from "react-router-dom";
-import ReactPlayer from "react-player";
```
Comment (ericboucher) [inline packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx:15]: are we using react-player anywhere else? If not maybe we can get of it as a dependency

Comment (tleraj) [reply packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx:15]: We use it in the About and FAQ pages

File: packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx (hunk 1 above)
Comment (tleraj) [reply packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx:36]: The goal is to use Material UI components as much as possible.

**Review by tleraj — APPROVED** [review]
Looks good!!

File: packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx (hunk 2 above)
Comment (ericboucher) [reply packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx:15]: should we use CardMedia there as well then?

Comment (tleraj) [reply packages/website/src/common/SiteDetails/FeaturedMedia/index.tsx:15]: I could try it yes 🙂

File: packages/website/src/common/SiteDetails/index.tsx [hunk 3]
```diff
@@ -72,61 +73,118 @@ const SiteDetails = ({
...
+              isoDate: surveyDiveDate,
+              format: "MMM DD[,] YYYY",
+              displayTimezone: false,
+              timeZone: reef.timezone,
+            })}`,
+            variant: "subtitle2",
+            marginRight: 0,
+          },
+        ];
+      case !!videoStream:
```
Comment (ericboucher) [inline packages/website/src/common/SiteDetails/index.tsx:101]: This case should come first

File: packages/website/src/common/SiteDetails/index.tsx [hunk 4]
```diff
@@ -72,61 +73,118 @@ const SiteDetails = ({
...
-    <Box mt="1rem">
-      {(!surveyDiveDate || !featuredSurveyPoint) && (
-        <CardTitle values={mapTitleItems} />
-      )}
+  const featuredMediaTitle = featuredMediaTitleItems();
 
-      <Grid container justify="space-between" spacing={2}>
-        <Grid item xs={12} md={6}>
-          {surveyDiveDate && featuredSurveyPoint && (
+  return (
```
Comment (ericboucher) [inline packages/website/src/common/SiteDetails/index.tsx:116]: This return statement suddenly got quite complicated. Can we try to clean it up a bit / make it a bit more future-proof?

**Review by ericboucher — APPROVED** [review]
